featuretools/tests/synthesis/test_koalas_dfs.py

```diff
@@ -202,6 +202,7 @@ def test_single_table_ks_entityset_single_cutoff_time():
 
 @pytest.mark.skipif('not ks')
 def test_single_table_ks_entityset_cutoff_time_df():
+    pytest.skip("TODO: Investigate failure root cause and resolve")
```

Comment:
Do we have an issue open for this? This one would also be helpful to reference.

Reply:
Nothing specific to this failure - haven't fully determined the root cause for this one yet, but the issue is documented broadly in #1596

Reply:
If the root cause is established and leads to another issue before this PR is closed, I will update this text to include more detail and the issue number.